File: C++/Level1/Hide Phone Number.cpp
/*
	https://programmers.co.kr/learn/courses/30/lessons/12948
*/

#include <string>

using namespace std;

string solution(string phone_number) {
	return phone_number.replace(phone_number.begin(), phone_number.end() - 4, phone_number.length() - 4, '*');
}

File: Java/Level1/HidePhoneNumber.java
/*
 * https://school.programmers.co.kr/learn/courses/30/lessons/12948
 */

class Solution {
    public String solution(String phone_number) {
        String substr = phone_number.substring(phone_number.length() - 4);
        String masking = new String("*").repeat(phone_number.length() - 4);

        return masking + substr;
    }
}